package.json

```diff
@@ -31,6 +31,6 @@
   "devDependencies": {
     "docsify-cli": "^4.4.3",
     "husky": "^6.0.0",
-    "lerna": "^4.0.0"
+    "lerna": "^3.22.1"
```

Comment:
Doesn't it also require an ignore update rule for renovate bot?

Reply:
yes but I'm usually quick at ignoring PRs when they come in, so I'd like to keep this repo able to upgrade once the lerna issue is solved. the problem with the ignore rule is that we'll forget it and we'll never upgrade